server/bin/pbench-sync-satellite.sh

```diff
@@ -250,10 +253,26 @@ for host in $hosts ;do
         for x in $state_list; do
             echo "$host/TO-SYNC/$x" >> ${state_change_log}
         done
+        rm $logdir/$host/ok-checks.log
+    fi
+
+    if [[ ! -s $logdir/$host/fail-checks.log ]] ; then
+        rm $logdir/$host/fail-checks.log
+    fi
+
+    if [[ -z $(grep -E -v -m 1 'OK|FAILED' $logdir/$host/md5-checks.log) ]] ; then
+        rm $logdir/$host/md5-checks.log
     fi
+
     popd > /dev/null 2>&4
 done
 
+rmdir ${logdir}/${host} 2>/dev/null
```

Comment:
Let's move this inside the `for host in ${hosts}; do` loop before line 267.

Reply:
And just the `rmdir` needs to move, the check for success or failure can be dropped because failure just means the directory contains log files we want to keep.